mantidproject/mantid
# Maxent algorithm output of iteration records and use of complex data

Ensure that Maxent algorithm returns for each spectrum, in the two workspaces that record the iterations exactly one element per iteration with no padding with zeros. Update the documentation and provide a usage example covering this and also how to combine two workspaces with complex data.

**To test:**

Build the documentation check that it is OK and up to date.

Run the usage example added to the documentation. 
Then select workspaces `EvolAngleP` & `EvolAngleP2` and plot them, then plot `EvolAngleC` both plots should be identical, except for legend and the second spectrum's plot should not go all the way to the right.

You may then try out other things with Maxent.

Fixes #21444. 

**Release Notes** 
In ...\release\v3.13.0\framework.html of documentation built by you.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
